script/buildSQLGeom.py

```diff
@@ -0,0 +1,236 @@
+#!/usr/bin/env python
+
+"""
+This module will accept an input text file with (x,y,[z]) coordinates in a CSV-format list
+and generate a SQL string that will add these coordinates as a PostGIS Polygon
+to sites.geometry within a BETYdb instance.
+
+A header row should be included in the input file, although the column names do not matter. If a z column
+for altitude is not included, the script will attempt to get it from USGS NED (US) or Google (global).
```

Comment:
In the current code, *everything* will go through USGS. My thinking was that if you have a valid key, we could just make everything go through Google. Do you consider USGS the preferred authority in the US?

We can make USGS domestic default and Google international default, but we need a good way to determine whether the point is in the US to begin with, which is likely another API call. Perhaps the anticipated traffic is low enough that this is fine. 

Another option would be to query USGS, and if that doesn't find anything, we query Google. Or, we could define a simple rectangle around the US and quickly check whether the lat/lon exceed that boundary, but that falls apart e.g. in northern Mexico. Don't love that idea.